Tests/test_image_access.py

```diff
@@ -363,8 +363,8 @@ def test_p_putpixel_rgb_rgba(self, mode):
 
 
 class TestImagePutPixelError(AccessTest):
-    IMAGE_MODES1 = ["L", "LA", "RGB", "RGBA"]
-    IMAGE_MODES2 = ["I", "I;16", "BGR;15"]
+    IMAGE_MODES1 = ["LA", "RGB", "RGBA"]
+    IMAGE_MODES2 = ["L", "I", "I;16"]
```

Comment:
Later in this file, `IMAGE_MODES2` are tested to raise "color must be int or single-element tuple", while `IMAGE_MODES1` are tested to raise "color must be int or tuple". This makes sense, as all `IMAGE_MODES2` are single channel, while `IMAGE_MODES1` have multiple channels.